p2p/sql_stmt.go

```diff
@@ -48,24 +70,167 @@ func (srv *Server) initSql() error {
 		if err := srv.prepareGetRowID(); err != nil {
 			return err
 		}
-	} else {
-		log.Trace("No sql db connection info provided")
 	}
 	return nil
 }
 
+func (srv *Server) checkIfTableExists(dbName string, tableName string) (int, error) {
+	var result int
+	err := srv.DB.QueryRow(`
+		SELECT COUNT(*) 
+		FROM information_schema.TABLES 
+		WHERE (TABLE_SCHEMA = ?) AND (TABLE_NAME = ?)
+	`, dbName, tableName).Scan(&result)
+	return result, err
```

Comment:
Why not just have this return a bool, err instead of an int, err?

Reply:
There 3 types of expected outcomes from this query: error, zero, and non-zero.
Error means the query failed implying there is something wrong with the sql server. In this case, we want the client exit right away.
Zero means the query didn't find any table with the name specified. In this case, the client should skip the rest of `backupTable` and continue running, as `createTables` will create the table anyways.
Non-zero (should be always 1) means the query found the table. This is the normal case.